arena.py

```diff
@@ -53,6 +53,12 @@ def bought_champion(self, name: str, slot: int) -> None:
         mk_functions.move_mouse(screen_coords.DEFAULT_LOC.get_coords())
         sleep(0.5)
         self.fix_bench_state()
+    
+    def force_bought_champion(self, name: str) -> None:
+        """Upgrade Champ when bench is full"""
+        mk_functions.move_mouse(screen_coords.DEFAULT_LOC.get_coords())
+        sleep(0.5)
+        self.fix_bench_state()
```

Comment:
What does this function do? The name is passed in as a parameter but isn't used.